interfaces/builtin/raw_usb.go

```diff
@@ -36,6 +36,7 @@ const rawusbConnectedPlugAppArmor = `
 
 # Allow access to all ttyUSB devices too
 /dev/ttyUSB[0-9]* rw,
+/dev/ttyACM[0-9]* rw,
```

Comment:
Reminds me of working on some cheap ass STM32 based devices using reference USB driver from ST.